sql-statements/sql-statement-overview.md

```diff
@@ -0,0 +1,300 @@
+| [`ALTER RESOURCE GROUP`](/sql-statements/sql-statement-alter-resource-group.md) | Modifies a resource group. |
+| [`CREATE RESOURCE GROUP`](/sql-statements/sql-statement-create-resource-group.md) | Creates a new resource group. |
+| [`DROP RESOURCE GROUP`](/sql-statements/sql-statement-drop-resource-group.md) | Drops a resource group. |
+| [`QUERY WATCH`](/sql-statements/sql-statement-query-watch.md) | Manage runaway query watch list. |
+| [`SET RESOURCE GROUP`](/sql-statements/sql-statement-set-resource-group.md) | Sets a resource group. |
+| [`SHOW CREATE RESOURCE GROUP`](/sql-statements/sql-statement-show-create-resource-group.md) | Shows the CREATE statement for a resource group. |
+
+</CustomContent>
+
+## Utility statements
+
+| SQL Statement | Description |
+|---------------|-------------|
+| [`DESC`](/sql-statements/sql-statement-desc.md) | Describes the structure of a table. |
+| [`DESCRIBE`](/sql-statements/sql-statement-describe.md) | Describes the structure of a table. |
+| [`DO`](/sql-statements/sql-statement-do.md) | Executes an expression. |
+| [`EXPLAIN`](/sql-statements/sql-statement-explain.md) | Shows the execution plan of a query. |
+| [`TRACE`](/sql-statements/sql-statement-trace.md) | Traces query execution. |
+| [`USE`](/sql-statements/sql-statement-use.md) | Sets the current database. |
+
+## Show statements
+
+| SQL Statement | Description |
+|---------------|-------------|
+| [`SHOW BUILTINS`](/sql-statements/sql-statement-show-builtins.md) | Lists builtin functions. |
+| [`SHOW CHARACTER SET`](/sql-statements/sql-statement-show-character-set.md) | Lists character sets. |
+| [`SHOW COLLATIONS`](/sql-statements/sql-statement-show-collation.md) | Lists collations. |
+| [`SHOW ERRORS`](/sql-statements/sql-statement-show-errors.md) | Shows error messages. |
+| [`SHOW STATUS`](/sql-statements/sql-statement-show-status.md) | Shows server status. |
+| [`SHOW VARIABLES`](/sql-statements/sql-statement-show-variables.md) | Shows system variables. |
+| [`SHOW WARNINGS`](/sql-statements/sql-statement-show-warnings.md) | Shows warning messages. |
+
+## Instance management
+
+<CustomContent platform="tidb">
+
+| SQL Statement | Description |
+|---------------|-------------|
+| [`ALTER INSTANCE`](/sql-statements/sql-statement-alter-instance.md) | Modifies an instance. |
+| [`FLUSH STATUS`](/sql-statements/sql-statement-flush-status.md) | Resets status variables. |
+| [`KILL`](/sql-statements/sql-statement-kill.md) | Kills a process. |
+| [`SHOW CONFIG`](/sql-statements/sql-statement-show-config.md) | Shows the configuration. |
+| [`SHOW ENGINES`](/sql-statements/sql-statement-show-engines.md) | Shows available storage engines. |
+| [`SHOW PLUGINS`](/sql-statements/sql-statement-show-plugins.md) | Shows installed plugins. |
+| [`SHOW PROCESSLIST`](/sql-statements/sql-statement-show-processlist.md) | Shows active threads. |
+| [`SHOW PROFILES`](/sql-statements/sql-statement-show-profiles.md) | Shows query profiles. |
```

Comment:
```suggestion
| [`SHOW PROFILES`](/sql-statements/sql-statement-show-profiles.md) | Currently only returns an empty result. |
```

Reply:
I think we should add `Included for compatibility with MySQL.` to this.